.github/workflows/tfplugindocs-generate.yml

```diff
@@ -0,0 +1,48 @@
+name: tfplugindocs generate
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/tfplugindocs-check.yml
+      - Makefile
+      - docs/**
+      - examples/**
+      - templates/**
+      - sonarqube/**
+
+jobs:
+  tfplugindocs_generate:
+    name: tfplugindocs generate
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.1
```

Comment:
lets be consistent and always use the git SHA in these github actions ref. Similar to how you have done in the tfplugin-check.yml

Reply:
Good catch @jdamata. I've updated these to use git SHA and retested on my fork to confirm everything still works as expected. 

Republished this PR - forgot I'd moved it back to draft whilst I was working on the changes.